aws/sqs/sqs.go

```diff
@@ -204,6 +205,38 @@ func (s *SQS) SendFifoMessage(queue, group, dedupe string, msg []byte) (string,
 	return "", nil
 }
 
+// Leverage the sendbatch api for uploading large numbers of messages
+func (s *SQS) SendBatch(queueURL string, Bodies []string, ctx context.Context) error {
+	var (
+		BodiesLen = len(Bodies)
+		maxlen    = 10
+		times     = BodiesLen / maxlen
+	)
+	var err error
+	var output *sqs.SendMessageBatchOutput
+	for i := 0; i < times; i++ {
+		var bodies = Bodies[maxlen*i : maxlen*(i+1)]
+		entries := make([]types.SendMessageBatchRequestEntry, len(bodies))
+		for j, body := range bodies {
+			entries[j] = types.SendMessageBatchRequestEntry{
+				Id:          aws.String(fmt.Sprintf("gamitjob%d", i*j)),
+				MessageBody: aws.String(body),
+			}
+		}
+		output, err = s.client.SendMessageBatch(ctx, &sqs.SendMessageBatchInput{
+			Entries:  entries,
+			QueueUrl: &queueURL,
+		})
```

Comment:
I think I need to check each message individually anyway from what's in the function docs 
https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/sqs@v1.18.3#Client.SendMessageBatch

> Because the batch request can result in a combination of successful and unsuccessful actions, you should check for batch errors even when the call returns an HTTP status code of 200

Reply:
okay I'll fail fast instead.